tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,144 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/versions/r1.1/programmers_guide/debugger). For instance, this dashboard
+
+* enables users to pause execution at specified ops or continue execution for some number of steps.
+* visualizes values of tensors.
+* associates tensors with specific lines in python code.
+
+![The Debugger Dashboard](images/debugger_intro.png)
+
+# Setup
+
+## Instrumenting Model Logic
+
+To use this dashboard, construct a `TensorBoardDebugWrapperSession`. Subsequently, that wrapper session will issue [gRPC](https://grpc.io/docs/guides/) messages to TensorBoard that containing data for debugging.
```

Comment:
Feel free to defer, but it would be good to point out that for users who use tf-learn's `Estimator`s and `Experiment`s,  `MonitoredSession`s, we provide `TensorBoardDebugHook`. Also point out that there are ways to debug Keras and tf-slim and refer to the aforementioned tfdbg doc link.

Reply:
Done.